exercises/minesweeper/example.js

```diff
@@ -0,0 +1,46 @@
+var Minesweeper = function() {
+  this.distanceXdistanceYs = [
+      [-1,-1],
+      [-1,0],
+      [-1,1],
+
+      [1,1],
+      [1,0],
+      [1,-1],
+
+      [0,1],
+      [0,-1]
+
+  ]
+};
+
+
+Minesweeper.prototype.annotate = function(rows) {
+  if(rows.length === 0 || rows[0].length === 0) {
+    return rows;
+  }
+  var board = rows.map(function(row) { return row.split(""); });
+  var outBoard = [];
+  for(var x = 0; x < board.length; x++) {
+    outBoard[x] = [];
+    for(var y = 0; y < board[x].length; y++) {

```

Comment:
I think a space between `for` and `(` would be conventional.  Both lines 24 and 26.